src/avatar.cpp

```diff
@@ -965,6 +1023,47 @@ void avatar::do_read( item &book )
         }
     }
 
+    // NPCs can't learn martial arts from manuals (yet).
+    auto m = book.type->use_methods.find( "MA_MANUAL" );
+    if( m != book.type->use_methods.end() ) {
+
+        const matype_id style_to_learn = martial_art_learned_from( *book.type );
+
+        skill_id skill_used = style_to_learn->primary_skill;
+        int difficulty = style_to_learn->learn_difficulty;
+        difficulty = 20 + difficulty * 2 - g->u.get_skill_level( skill_used ) * 2;
```

Comment:
Might want to clamp this below at 1.  If `difficulty` becomes negative strange things might happen?